spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,119 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::CommentsController, type: :controller do
+  let(:user) { create(:user) }
+  let(:project) { create(:project, user: user) }
+  let!(:task) { create(:task, project: project) }
```

Comment:
Maybe it would be better to use some traits such as `create(:user, :with_project)` or even `create(:user, :with_tasks)`?